sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -385,34 +385,42 @@ def list_accelerators(
             new_results[acc_name] = acc_info
     results = new_results
 
-    # Unlike other GPUs that can be attached to different sizes of N1 VMs,
-    # A100 GPUs can only be attached to fixed-size A2 VMs,
-    # and L4 GPUs can only be attached to G2 VMs.
-    # Thus, we can show their exact cost including the host VM prices.
-
-    acc_infos: List[common.InstanceTypeInfo] = sum(
-        [results.get(a, []) for a in _ACC_INSTANCE_TYPE_DICTS], [])
-    if not acc_infos:
-        return results
+    # Figure out which instance type to use.
+    infos_with_instance_type: List[common.InstanceTypeInfo] = sum(
+        results.values(), [])
 
     new_infos = defaultdict(list)
-    for info in acc_infos:
-        assert pd.isna(info.instance_type) and pd.isna(info.memory), acc_infos
-        vm_types = _ACC_INSTANCE_TYPE_DICTS[info.accelerator_name][
-            info.accelerator_count]
+    for info in infos_with_instance_type:
+        assert pd.isna(info.instance_type) and pd.isna(
+            info.memory), infos_with_instance_type
```

Comment:
```suggestion
            info.memory), infos
```
Should we only print `infos` here instead of the whole list?